packages/devtools_app/lib/src/app.dart

```diff
@@ -581,8 +581,8 @@ List<DevToolsScreen> get defaultScreens {
     DevToolsScreen<InspectorController>(
       InspectorScreen(),
       createController: (_) => InspectorController(
-        inspectorTree: InspectorTreeController(),
-        detailsTree: InspectorTreeController(),
+        inspectorTree: InspectorTreeController(gaId: 'inspectorTree'),
+        detailsTree: InspectorTreeController(gaId: 'detailsTree'),
```

Comment:
make these strings constants on the InspectorTreeControllerMetrics class since that is where they are used. Or make them an enum if they will be used in more places than just here